yb-voyager/cmd/exportData.go

```diff
@@ -1353,6 +1353,12 @@ func clearMigrationStateIfRequired() {
 			utils.ErrExit("Failed to remove name registry file: %s", err)
 		}
 
+		err = metaDB.UpdateMigrationStatusRecord(func(record *metadb.MigrationStatusRecord) {
+			record.TableListExportedFromSource = nil
```

Comment:
what about the Target* fields? start-clean export-data means everything starts afresh, so I guess we should clean those fields up  as well?

Reply:
it's not required to do this as there is no way after the cutover to target to do `export data --start-clean t` as they have to start a new migration altogether but still I have added a few more to this.

Reply:
I think we might be already using or need this for other commands also.
can we organise this separately


```
metaDB.UpdateMigrationStatsRecord_StartCleanExportSchema()
metaDB.UpdateMigrationStatsRecord_StartCleanExportData()
metaDB.UpdateMigrationStatsRecord_StartCleanImportSchema()
metaDB.UpdateMigrationStatsRecord_StartCleanImportData()...
```

Reply:
Can we do this in a separate PR after analyzing all other Fields in MSR we want to cleanup for other commands? If so I create a ticket and will pick it up separately.

Reply:
yeah sure. 
Maybe create a new ticket and link it with tech debt one you created the other day.